sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/AuthClient.java

```diff
@@ -0,0 +1,184 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.jca;
+
+import com.azure.security.keyvault.jca.rest.OAuthToken;
+
+import java.util.HashMap;
+import java.util.logging.Logger;
+
+import static java.util.logging.Level.FINER;
+
+/**
+ * The REST client specific to getting an access token for Azure REST APIs.
+ *
+ * @author Manfred Riem (manfred.riem@microsoft.com)
```

Comment:
We don't include author tags in our JavaDoc.

Reply:
I will address this